### PR TITLE
Change R Dependencies

### PR DESCRIPTION
**Pull request by pablo-gar**
- Changes Seurat and SingleCellExperiment from depends to suggest. 
- Updates install instructions

### CODE REVIEW

File: api/r/cellxgene.census/R/get_helpers.R [hunk 1]
```diff
@@ -135,6 +140,10 @@ get_single_cell_experiment <- function(
     var_coords = NULL,
     var_column_names = NULL,
     var_index = "feature_id") {
+  stopifnot(
+    !require("SingleCellExperiment"),
+    "R package 'Seurat' is not installed."
+  )
```
Comment (pablo-gar) [inline api/r/cellxgene.census/R/get_helpers.R:146]:
```suggestion
  )
  
```

File: api/r/cellxgene.census/R/get_helpers.R [hunk 2]
```diff
@@ -69,6 +69,11 @@ get_seurat <- function(
     var_coords = NULL,
     var_column_names = NULL,
     var_index = "feature_id") {
+  stopifnot(
+    !require("Seurat"),
```
Comment (pablo-gar) [inline api/r/cellxgene.census/R/get_helpers.R:73]:
```suggestion
    require("Seurat"),
```

File: api/r/cellxgene.census/R/get_helpers.R [hunk 3]
```diff
@@ -135,6 +140,11 @@ get_single_cell_experiment <- function(
     var_coords = NULL,
     var_column_names = NULL,
     var_index = "feature_id") {
+  stopifnot(
+    !require("SingleCellExperiment"),
```
Comment (pablo-gar) [inline api/r/cellxgene.census/R/get_helpers.R:144]:
```suggestion
    require("SingleCellExperiment"),
```

File: api/r/cellxgene.census/R/get_helpers.R [hunk 4]
```diff
@@ -135,6 +140,11 @@ get_single_cell_experiment <- function(
     var_coords = NULL,
     var_column_names = NULL,
     var_index = "feature_id") {
+  stopifnot(
+    require("SingleCellExperiment"),
+    "R package 'Seurat' is not installed."
```
Comment (pablo-gar) [inline api/r/cellxgene.census/R/get_helpers.R:145]:
```suggestion
    "R package 'SingleCellExperiment' is not installed."
```

File: docs/cellxgene_census_docsite_installation.md [hunk 5]
```diff
@@ -29,11 +29,13 @@ If installing in a Databricks notebook environment, use `%pip install`. Do not u
 
 ## R
 
-From an R session, first install `tiledb` from R-Universe, the latest release in CRAN is not yet available.
+If installing from **Ubuntu**, you may need to install the following libraries via `apt install`,  `libxml2-dev` `libssl-dev` `libcurl4-openssl-dev`. In addition you must have `camke` v3.21 or greater.
```
Comment (mlin) [inline docs/cellxgene_census_docsite_installation.md:32]:
```suggestion
If installing from **Ubuntu**, you may need to install the following libraries via `apt install`,  `libxml2-dev` `libssl-dev` `libcurl4-openssl-dev`. In addition you must have `cmake` v3.21 or greater.
```

**Review by mlin — APPROVED** [review]
see inline typo fixes

**Pull request merged**